File: 8-kyu/grasshopper---grade-book/python/solution.py
def get_grade(s1, s2, s3):
    grades = [s1, s2, s3]
    avg = sum(grades) / len(grades)
    if 90 <= avg <= 100: return 'A'
    elif 80 <= avg < 90: return 'B'
    elif 70 <= avg < 80: return 'C'
    elif 60 <= avg < 70: return 'D'
    return 'F'
    

File: 8-kyu/grasshopper---grade-book/c/solution.c
char get_grade(int a, int b, int c)
{
    int avg = (a + b + c) / 3;
    if (avg < 60) return 'F';
    else if (avg < 70) return 'D';
    else if (avg < 80) return 'C';
    else if (avg < 90) return 'B';
    else return 'A';
}